rust/rsc/src/bin/rsc/config.rs

```diff
@@ -15,8 +15,20 @@ pub struct RSCTTLConfig {
     pub tick_rate: u64,
     // How long an object is allowed to live
     pub ttl: u64,
-    // Maximum number of objects to delete at a time. Must be 1 >= x <= 16000
+    // Maximum number of objects to delete from the db at a time. Must be 1 >= x <= 16000
+    pub chunk_size: u32,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct RSCBlobTTLConfig {
+    // How often to run the eviction check in seconds
+    pub tick_rate: u64,
+    // How long an object is allowed to live
+    pub ttl: u64,
+    // Maximum number of objects to delete from the db at a time. Must be 1 >= x <= 16000
     pub chunk_size: u32,
+    // Maximum number of files to delete from the disk per task
+    pub file_chunk_size: usize,
```

Comment:
Is `usize` always going to be less than `u32`?

Reply:
no `usize` is the number of bits a pointer on the host has (so pretty much `u64` everywhere). The API that the value goes into expects a `usize` so it was easier to let the config system do validation instead  of pushing it all the way into the eviction task

Reply:
Sure thats fine.